src/components/Toasts/TransactionToast.styles.tsx

```diff
@@ -0,0 +1,36 @@
+import styled from "styled-components/macro";
+
+export const Container = styled.div`
+  color: ${(props) => props.theme.colors.white};
```

Comment:
Try to order your styling according to the box model. For instance `color` and `background` don't have any effect on the size / position of your, but `display` and `grid-template-columns` do.

```
  display: grid;
  grid-template-columns: auto auto auto;
  grid-gap: 1rem;
  justify-items: center;
  align-items: center;
  padding: 1rem;
  color: ${(props) => props.theme.colors.white};
  background: ${(props) => props.theme.colors.black};
```